ansible/roles/data-products-deploy/defaults/main.yml

```diff
@@ -20,7 +20,7 @@ job_manager_tmp_dir: "transient-data"
 channel: dev-test
 druid_broker_host: "{{groups['broker'][0]}}"
 
-analytics_job_list: '"wfs", "ds", "dpu", "content-rating-updater", "daily-metrics", "desktop-consumption-report", "course-enrollment-report", "audit-metrics-report", "monitor-job-summ"'
+analytics_job_list: '"wfs", "ds", "dpu", "content-rating-updater", "daily-metrics", "desktop-consumption-report", "course-enrollment-report", "etb-metrics", "audit-metrics-report", "monitor-job-summ"'
 analytics_jobs_count: 9
```

Comment:
Need to update this count variable according to job_list. Please update it

Reply:
Done